src/Microsoft.ComponentDetection/Program.cs

```diff
@@ -32,6 +32,9 @@
 
     Console.WriteLine($"Execution finished, status: {exitCode}.");
 
+    // Manually dispose to flush logs as we force exit
+    await serviceProvider.DisposeAsync();
```

Comment:
`await using` instead of manually calling dispose?

Reply:
We cannot use `await using` as dispose will not be called when we call `Environment.Exit`